frontend/css/home_view.css

```diff
@@ -2,9 +2,70 @@ body {
     width: 99%;
 }
 
-.row {
-    margin-right: -15px;
-    margin-left: -15px;
+/* Extra small devices (portrait phones, less than 576px)*/
+@media (max-width: 575.98px) {
+    .responsive-div {
+        flex-basis: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=6343113)